File: src/main/java/io/github/bot/terminal/application/domains/notificarion_api/values/NotificationApiType.java
package io.github.bot.terminal.application.domains.notificarion_api.values;

public enum NotificationApiType {

    TELEGRAM_BOT("Telegram Bot API"),
    SLACK_BOT("Slack Bot API");

    private String label;

    NotificationApiType(String label) {
        this.label = label;
    }
    public String getLabel() {
        return label;
    }
    public interface Constants {
        String TELEGRAM_BOT = "TELEGRAM_BOT";
        String SLACK_BOT = "SLACK_BOT";
    }
}

File: src/main/java/io/github/bot/terminal/application/domains/notificarion_api/values/NotificationApiType.kt
package io.github.bot.terminal.application.domains.notificarion_api.values

enum class NotificationApiType(val label: String) {
    TELEGRAM_BOT("Telegram Bot API"),
    SLACK_BOT("Slack Bot API");

    interface Constants {
        companion object {
            const val TELEGRAM_BOT = "TELEGRAM_BOT"
            const val SLACK_BOT = "SLACK_BOT"
        }
    }

}